ci-operator/config/openshift/console/openshift-console-master.yaml

```diff
@@ -63,6 +63,26 @@ tests:
   commands: ./test-prow-e2e.sh
   openshift_installer_src:
     cluster_profile: gcp
+- artifact_dir: /tmp/artifacts
+  as: kubevirt-plugin
+  commands: |
+    export ARTIFACT_DIR=/tmp/artifacts
+
+    # install virtctl
```

Comment:
can we move virtctl installation also into deploy-kubevirt-gating.sh so the logic for deploying kubevirt would be in one place? Plus it would use the same version variable.

Reply:
I thought about it, but then the `export` wouldn't work from inside the script. 
the solution of course is to split it but that no so good either since a change in this `virtctl` section would need to be done In two different places - `openshift/release` and `openshift/console` I'd like to avoid 2 PRs in 2 different repos for 1 small change in the future.